MAUI/DataForm/migration.md

```diff
@@ -0,0 +1,988 @@
+---
+layout: post
+title: Migrate from Xamarin to .Net MAUI DataForm (SfDataForm) | Syncfusion
+description: Learn here all about Migrating from Syncfusion Xamarin.Forms DataForm to Syncfusion .NET MAUI DataForm control.
+platform: MAUI
+control: SfDataForm
+documentation: ug
+---  
+
+# Migrate from Xamarin.Forms SfDataForm to .NET MAUI SfDataForm 
+
+To make the migration from the Xamarin [Xamarin SfDataForm] to [.NET MAUI SfDataForm] easier, most of the APIs from the Xamarin SfDataForm were kept in the.NET MAUI SfDataForm. However, to maintain the consistency of API naming in the.NET MAUI SfDataForm, some of the APIs have been renamed. Please find the difference in the following topics.
```

Comment:
To make the migration from the  [Xamarin SfDataForm](https://www.syncfusion.com/xamarin-ui-controls/xamarin-dataform)

add link - https://www.syncfusion.com/xamarin-ui-controls/xamarin-dataform

Reply:
[.NET MAUI SfDataForm] => `.NET MAUI SfDataForm`

Replace all applicable places

Reply:
<img width="86" alt="image" src="https://user-images.githubusercontent.com/92788387/208393267-2b11a037-b5c1-43d9-a386-233a29659dec.png">

space issue

Reply:
changes done